generated/nidcpower/nidcpower/session.py

```diff
@@ -4869,11 +4882,24 @@ def measure_multiple(self):
 
         '''
         import collections
-        Measurement = collections.namedtuple('Measurement', ['voltage', 'current', 'in_compliance'])
+        Measurement = collections.namedtuple('Measurement', ['voltage', 'current', 'in_compliance', 'channel'])
 
         voltage_measurements, current_measurements = self._measure_multiple()
 
-        return [Measurement(voltage=voltage_measurements[i], current=current_measurements[i], in_compliance=None) for i in range(self._parse_channel_count())]
+        channel_names = _converters.convert_channels_repeated_capabilities(
+            self._repeated_capability,
+            session_channel_names=self._get_channel_names(range(self._parse_channel_count()))
```

Comment:
It'd be nice if we stored this in the Session (and pass it to SessionBase) objects so we're not creating this list on every function call.

`self._all_channels_in_session`

Reply:
if we do this... do we still `_get_channel_names` to be accessible from SessionBase? I think no!

Reply:
Populating `self._all_channels_in_session` can be done in the shared template for all modules, surrounded by `try`/`except`.